keyboards/vn66/rules.mk

```diff
@@ -34,12 +34,21 @@ COMMAND_ENABLE = yes        # Commands for debug and configuration
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = no            # USB Nkey Rollover
+<<<<<<< HEAD:keyboards/vn66/rules.mk
```

Comment:
You have some merge artifacts here. 

You should be able to run `git checkout upstream/master keyboards/vn66/rules.mk` to fix this.